src/components/LuxMenuBar.vue

```diff
@@ -667,7 +668,6 @@ export default {
   background: var(--color-white);
   a,
   .lux-submenu-toggle {
-    &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}